apps/web/server/routers/viewer/slots.tsx

```diff
@@ -198,8 +198,8 @@ export const slotsRouter = createRouter().query("getSchedule", {
         periodDays: eventType.periodDays,
       });
 
-    // AFAIK startTime and endTime already have timezone info on them
-    let time = dayjs(startTime);
+    let time = input.timeZone === "Etc/GMT" ? startTime.utc() : startTime.tz(input.timeZone);
```

Comment:
@zomars Alex change here was right, as the timezone it's only being received in the input.